x-pack/filebeat/input/awss3/metrics.go

```diff
@@ -25,9 +25,12 @@ type inputMetrics struct {
 	sqsMessageProcessingTime            metrics.Sample   // Histogram of the elapsed SQS processing times in nanoseconds (time of receipt to time of delete/return).
 
 	s3ObjectsRequestedTotal *monitoring.Uint // Number of S3 objects downloaded.
-	s3ObjectsAckedTotal     *monitoring.Uint // Number of S3 objects fully ACKed.
-	s3ObjectsListedTotal    *monitoring.Uint // Number of S3 objects listed.
-	s3ObjectsProcessedTotal *monitoring.Uint // Number of S3 objects processed.
+	// s3ObjectsListedTotal is the number of S3 objects processed that were fully ACKed.
+	s3ObjectsAckedTotal *monitoring.Uint // Number of S3 objects fully ACKed.
```

Comment:
This one has two godocs.
```suggestion
	s3ObjectsAckedTotal *monitoring.Uint
```